services/QuillLMS/app/models/activity.rb

```diff
@@ -227,9 +243,24 @@ def add_question(question)
   end
 
   def readability_grade_level
-    return nil unless raw_score_id
+    raw_score&.readability_grade_level(activity_classification_id)
+  end
+
+  def default_minimum_grade_level
+    return nil if readability_grade_level.nil?
+
+    READABILITY_GRADE_LEVEL_TO_MINIMUM_GRADE_LEVEL_MAP[readability_grade_level]
+  end
+
+  def default_maximum_grade_level
+    return nil if readability_grade_level.nil?
+
+    DEFAULT_MAX_GRADE_LEVEL
+  end
 
-    raw_score.readability_grade_level(activity_classification_id)
+  def set_minimum_and_maximum_grade_levels_to_default_values
+    self.minimum_grade_level = default_minimum_grade_level
+    self.maximum_grade_level = default_minimum_grade_level ? default_maximum_grade_level : nil
```

Comment:
self.maximum_grade_level = default_maximum_grade_level if default_minimum_grade_level

Reply:
So that's actually what I had initially, but I think in the case where for whatever reason there already was a `maximum_grade_level` set here, you would want to explicitly reset it to nil.